### ISSUE TITLE
cogvideo-controlnet 5b oom

### ISSUE THREAD

**Issue by linwenzhao1** [TheDenk/cogvideox-controlnet#24]
Hello, may I ask why I use two 80g cards to run the 5b model, and still have the oom problem. Here are the parameter Settings：
accelerate_config_machine_mutil.yaml:
compute_environment: LOCAL_MACHINE
main_process_port: 29501
debug: false
deepspeed_config:
  gradient_accumulation_steps: 1
  gradient_clipping: 1.0
  offload_optimizer_device: none
  offload_param_device: none
  zero3_init_flag: false
  zero_stage: 2
distributed_type: DEEPSPEED
downcast_bf16: 'no'
enable_cpu_affinity: false
machine_rank: 0
main_training_function: main
dynamo_backend: 'no'
mixed_precision: 'no'
num_machines: 1
num_processes: 2
rdzv_backend: static
same_network: true
tpu_env: []
tpu_use_cluster: false
tpu_use_sudo: false
use_cpu: false
main_process_port: 29079

finetune_multi_rank.sh
#!/bin/bash
export MODEL_PATH="/home/CogVideo/outputs_5b_1000"
export PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True
export CUDA_VISIBLE_DEVICES="6,7"

# if you are not using wth 8 gus, change `accelerate_config_machine_single.yaml` num_processes as your gpu number
accelerate launch --config_file accelerate_config_machine_multi.yaml --multi_gpu \
  train_controlnet.py \
  --tracker_name "cogvideox-controlnet-5b" \
  --gradient_checkpointing \
  --pretrained_model_name_or_path $MODEL_PATH \
  --enable_tiling \
  --enable_slicing \
  --validation_prompt "car is going in the ocean, beautiful waves:::ship in the vulcano" \
  --validation_video "../resources/car.mp4:::../resources/ship.mp4" \
  --validation_prompt_separator ::: \
  --num_inference_steps 28 \
  --num_validation_videos 1 \
  --validation_steps 500 \
  --seed 42 \
  --mixed_precision bf16 \
  --output_dir "cogvideox-controlnet-5b" \
  --height 480 \
  --width 720 \
  --fps 8 \
  --max_num_frames 41 \
  --video_root_dir "/home/train/videos/" \
  --csv_path "/home/train_anno.csv" \
  --stride_min 1 \
  --stride_max 3 \
  --hflip_p 0.5 \
  --controlnet_type "canny" \
  --controlnet_transformer_num_layers 4 \
  --controlnet_input_channels 3 \
  --downscale_coef 8 \
  --controlnet_weights 0.5 \
  --init_from_transformer \
  --train_batch_size 1 \
  --dataloader_num_workers 0 \
  --num_train_epochs 10000 \
  --checkpointing_steps 200 \
  --gradient_accumulation_steps 2 \
  --learning_rate 1e-5 \
  --lr_scheduler cosine_with_restarts \
  --lr_warmup_steps 250 \
  --lr_num_cycles 1 \
  --enable_slicing \
  --enable_tiling \
  --gradient_checkpointing \
  --optimizer AdamW \
  --adam_beta1 0.9 \
  --adam_beta2 0.95 \
  --max_grad_norm 1.0 \
  --allow_tf32
